### PR TITLE
JsonLayout - IncludeMdc and IncludeMdlc

### PR DESCRIPTION
**Pull request by snakefoot**
Partial fix for #2108 (Missing NDC)

### CODE REVIEW

**Comment by snakefoot** [comment]
Guess something is wrong with NuGet. Recommended workaround is to upgrade to NuGet 3.5 and amend the nuget restore command like this:

> appveyor-retry nuget restore

http://help.appveyor.com/discussions/problems/6660-nuget-packages-sporadically-failing-to-download

**Comment by 304NotModified** [comment]
thx, will do a restart for now

File: src/NLog/Internal/XmlHelper.cs [hunk 1]
```diff
@@ -126,15 +126,13 @@ internal static string XmlConvertToString(object value)
         /// <param name="value">Object value</param>
         /// <param name="objTypeCode">Object TypeCode</param>
         /// <returns>Object value converted to string</returns>
-        internal static string XmlConvertToString(object value, out TypeCode objTypeCode)
```
Comment (304NotModified) [inline src/NLog/Internal/XmlHelper.cs]: nice improvement :)

**Comment by snakefoot** [comment]
@304NotModified Updated Wiki https://github.com/NLog/NLog/wiki/JsonLayout

**Comment by 304NotModified** [comment]
👍 👍 @snakefoot

**Comment by aheubusch** [comment]
If IncludeMdc or IncludeMdlc is true then the layout is no longer ThreadAgnostic.

**Comment by 304NotModified** [comment]
I think you're right!

**Comment by 304NotModified** [comment]
Could you please create a new issue? It's OK to keep the description short. Thanks!

**Pull request merged**